### DISCUSSION TITLE
recolorable footpaths

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17760] [Feature requests] [answered]
Right now the footpaths menu allows you to select between different colored footpaths and queues, but for an enhancement, I think some of these should be replaced with a single recolorable footpath item
some of these paths:
![asddasadsdas](https://user-images.githubusercontent.com/94884086/183343484-0a279fef-ea4a-45f7-a912-91323b161095.png)

would change to:

![paths](https://user-images.githubusercontent.com/94884086/183344721-f2d7a402-ac44-478c-a184-ddd668008ef1.png)

The paths in the picture would be recolorable, and consolidated into 3 path options. The paths with checkerboard and two tone queues would have a primary and secondary color. In this way, we simplify the pathing menu, and add more color options for these paths. When you click on "Queue" for example, you could choose between the expansion or other fancy RCT1 queue types or the recolorable queue, then if you picked recolorable queue, the pathing menu would have an option for recoloring. Alternatively, this recoloring option could be grayed out when not available

**Comment by J-PIE-314** [comment, marked as answer]
This suggestion was made in part of #16035.

**Comment by 733737** [comment]
the developers have stated once paths are refactored they will add an additional queue style that is recolorable and keep the other queues for compatibility

**Reply by J-PIE-314** [reply]
As seen in #15889?
